### PR TITLE
Feature/support multiple sidebars

### PR DESCRIPTION
**Pull request by DaveHae**
**Why**
Add support for multiple sidebars

**How**
Add configuration keys:
- subfolders for sidebar content which is located in subfolders
- productTitles to display the name of the product/sidebar on the cover page and the header
- Reset slugger on every sidebar instead of version in case they have the same document name in order to not have "*-1.pdf" files

### CODE REVIEW

File: src/generate.ts [hunk 1]
```diff
@@ -111,32 +111,53 @@ export async function generatePdfFiles(
...
+      
+      if (versionInfo.urlAddIn) {
+        rootDocUrl = `${rootDocUrl}${versionInfo.urlAddIn}/`;
       }
-    }
 
-    // Loop through all configured sidebar names
-    for (const sidebarName of pluginOptions.sidebarNames) {
+      
+      if (versionInfo.urlAddIn && versionInfo.urlAddIn) {
```
Comment (techfreak01) [inline src/generate.ts]: This double check for same field seems to be a mistake.
Should there be another 2nd condition?

**Review by techfreak01 — CHANGES_REQUESTED** [review]
Thank you for the pull request with the new feature.
I modified the README a bit to not be confusing with the limitation about support of blog posts.

And I added a question about checking twice the same variable in if statement.

File: src/generate.ts (hunk 1 above)
Comment (DaveHae) [reply src/generate.ts]: Fixed the if statement. Now checks for versionInfo.urlAddIn and folderName

**Review by techfreak01 — APPROVED** [review]
Changes in last commit resolves my question.

**Pull request merged**